<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeasuredDataPointGroupDataItem_PropertyDialog</class>
 <widget class="QDialog" name="MeasuredDataPointGroupDataItem_PropertyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Points Display Setting Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="displaySettingGroupBox">
       <property name="title">
        <string>Display Setting</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QRadioButton" name="pointsRadioButton">
          <property name="text">
           <string>Points</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="surfaceRadioButton">
          <property name="text">
           <string>Surface</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="colorGroupBox">
     <property name="title">
      <string>Color</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,1">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="customColorRadioButton">
          <property name="text">
           <string>Custom Color</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="ColorEditWidget" name="colorEdit">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="valueRadioButton">
          <property name="text">
           <string>Color by value</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QComboBox" name="valueComboBox"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="colorMapGroupBox">
        <property name="title">
         <string>Colormap Setting</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
         <item>
          <widget class="ColorMapSettingEditWidget" name="colorMapWidget" native="true"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="importButton">
             <property name="text">
              <string>&amp;Import...</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="exportButton">
             <property name="text">
              <string>&amp;Export...</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="pointSizeLabel">
       <property name="text">
        <string>Point Size:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pointSizeSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="TransparencyWidget" name="transparencyWidget" native="true"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorEditWidget</class>
   <extends>QLabel</extends>
   <header>guibase/widget/coloreditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TransparencyWidget</class>
   <extends>QWidget</extends>
   <header>guibase/widget/transparencywidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ColorMapSettingEditWidget</class>
   <extends>QWidget</extends>
   <header>guicore/scalarstocolors/colormapsettingeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MeasuredDataPointGroupDataItem_PropertyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>1002</x>
     <y>1214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MeasuredDataPointGroupDataItem_PropertyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>803</x>
     <y>620</y>
    </hint>
    <hint type="destinationlabel">
     <x>1113</x>
     <y>1214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>valueComboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>customColorRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>100</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>valueRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorMapGroupBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>128</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pointsRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>pointSizeSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>550</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
